doc/NEWS-2.7.0

```diff
@@ -366,6 +366,15 @@ See also Warning in {Core classes updates}[#label-Core+classes+updates+-28outsta
 
 [File]
 
+  [New method]
+
+    * Added File.absolute_path? to check whether a path is absolute or
+      not in a portable way.
```

Comment:
I'd like to add a ticket number reference here.

```suggestion
      not in a portable way. [Feature #15868]
```

Reply:
Thanks for your suggestion! I committed it in https://github.com/ruby/ruby/pull/4139/commits/002b5f97fdf10c8ac26a98bffc6faf2e27fccafd